docs/src/Upsell.doc.js

```diff
@@ -251,4 +257,79 @@ function Example(props) {
   />,
 );
 
+card(
```

Comment:
Updated the doc using Table as an example! Let me know if there are any changes that should be made!

Reply:
Awesome! It looks like the props in the table need to be updated, though 🙂